### ISSUE TITLE
It would be nice to make it without the need of the TT2vga

### ISSUE THREAD

**Issue by Zibri** [rpsubc8/ESP32TinyC64#1]
Using this https://www.cnx-software.com/2022/03/28/esp32-composite-video-library-outputs-pal-secam-and-ntsc-supports-lvgl/

or this:
https://github.com/bitluni/ESP32CompositeVideo

and this:
https://github.com/rossumur/esp_8_bit

**Comment by rpsubc8** [comment]
Thank you for the information.
Some time ago, I modified the bitluni composite video cvbs library a bit so that it can also be used on the TTGO VGA32 board on GPIO 26 (pinout 5 CLK PS/2), as well as other boards:

https://github.com/rpsubc8/ESP32TinyChip8

https://github.com/rpsubc8/ESP32TinyMCUMEesp81

With more time, I'll be putting it on all the emulators I port, including this C64 one.

**Comment by Zibri** [comment]
Yep.. but I think it's WAY cooler to have the ESP32 do the job without any other "hat" or device attached.
